waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/Waimak.scala

```diff
@@ -49,6 +50,8 @@ object Waimak {
     *
     * @return
     */
-  def sparkExecutor(): DataFlowExecutor[SparkFlowContext] = new SequentialDataFlowExecutor[SparkFlowContext](SparkFlowReporter)
+  def sparkExecutor(): DataFlowExecutor[SparkFlowContext] = ParallelDataFlowExecutor[SparkFlowContext](SparkFlowReporter)(SparkFlowContext.setPoolIntoContext)
```

Comment:
I think I would prefer it if there was still just the one `sparkExecutor()` method, but which takes `maxJobs` and `priorityStrategy` as parameters which have default values. Thoughts?

Reply:
I also don't really think the default for `maxJobs` should be 1. I would prefer if it was parallel by default, and that sequential execution should be a conscious choice

Reply:
Also, maybe rename `maxJobs` to `maxThreads`?

Reply:
> Also, maybe rename `maxJobs` to `maxThreads`?

or `maxParallelJobs`, or `maxParallelActions`?

Reply:
maxParallelActions feels better, will do

Reply:
what the default should be?

Reply:
> what the default should be?

That's up for debate....maybe 20?

Reply:
done, def sparkExecutor(maxParallelActions: Int = 20, priorityStrategy: priorityStrategy[SparkFlowContext] = DFExecutorPriorityStrategies.raceToOutputs)